packages/expo-dev-menu/ios/EXDevMenuBuildInfo.h

```diff
@@ -0,0 +1,22 @@
+//
+//  DevMenuBuildInfo.h
+//  expo-dev-menu-EXDevMenu
+//
+//  Created by andrew on 2022-01-10.
+//
+
+#import <Foundation/Foundation.h>
+#import <React/RCTBridge.h>
+#import <EXManifests/EXManifestsManifestFactory.h>
```

Comment:
remove this as it looks unused? (if you want to use this import you will probably need to add EXManifests as a dependency in the expo-dev-menu podspec)